go/azidentity/azure_identity_credential_adapter.go

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package azidentity
```

Comment:
I'm also unsure how I feel about naming this package `azidentity`.  Is the assumption that folks will use this in lieu of the "real" `azidentity`?  If they want to use the adapter in tandem with some other credential mechanism in the "real" `azidentity` then one of the package names will have to be aliased; not terrible, but would indicate that we didn't think about this scenario.

Reply:
Should be azidext